features/application/Titles.feature

```diff
@@ -18,13 +22,13 @@ Feature: application / titles
       | /onboarding/dataplanes-overview | Data plane overview |
       | /onboarding/completed           | Completed           |
 
-      | /zones/create                           | Create & connect Zone |
-      | /zones/zone-cps                         | Zone Control Planes   |
-      | /zones/zone-cps/zone-cp-name            | zone-cp-name          |
-      | /zones/zone-ingresses                   | Ingresses             |
-      | /zones/zone-ingresses/zone-ingress-name | zone-ingress-name     |
-      | /zones/zone-egresses                    | Egresses              |
-      | /zones/zone-egresses/zone-egress-name   | zone-egress-name      |
+      | /zones/create                                            | Create & connect Zone |
+      | /zones/zone-cps                                          | Zone Control Planes   |
+      | /zones/zone-cps/zone-cp-name                             | zone-cp-name          |
```

Comment:
Should this be `overview`?

Reply:
You mean the URL here right? (lmk if not)

To be exact yes, but! `/zones/zone-cps/zone-cp-name` should redirect to `/zones/zone-cps/zone-cp-name/overview` (which it does), so these types of 'incomplete/redirect URL' tests are kinda nice to have.

e.g. In a slightly different test, when I thought I had finished this piece of work and then ran the tests a bunch failed because I hadn't added redirects in certain places

Reply:
Yeah gimme 1 sec I'm just rebasing this now that https://github.com/kumahq/kuma-gui/pull/1459 has gone in

Reply:
K thats rebased, gonna resolve this convo now as I think the rebase resolved it, let me know on the rest @kleinfreund